### PR TITLE
add toolBar. hide toolbar and bottom nav bar on AuthenticationFragmen…

### PR DESCRIPTION
**Pull request by JustAnOtherCoder13**
left to show drawer menu on toolbar and implement MapsFragment

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -6,6 +6,54 @@
...
     tools:context=".presentation.ui.MainActivity">
 
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?android:attr/actionBarSize"
+        android:background="@color/colorPrimary"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
```
Comment (fodeguirassy) [inline app/src/main/res/layout/activity_main.xml]: format

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -6,6 +6,54 @@
...
+                android:contentDescription="@string/toolbar_menu_icon_content_description_activity_main" />
+
+            <TextView
+                android:id="@+id/top_nav_bar_title_text_view"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_centerInParent="true"
+                android:text="@string/app_name"
+                android:textColor="@color/onPrimaryColor"
+                android:textStyle="bold"
```
Comment (fodeguirassy) [inline app/src/main/res/layout/activity_main.xml]: same here

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -26,6 +74,7 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
-        app:menu="@menu/bottom_nav_bar_menu" />
+        app:menu="@menu/bottom_nav_bar_menu"
```
Comment (fodeguirassy) [inline app/src/main/res/layout/activity_main.xml]: format

File: app/src/main/java/com/picone/go4lunch/presentation/ui/MainActivity.java [hunk 4]
```diff
@@ -21,9 +29,34 @@ protected void onCreate(Bundle savedInstanceState) {
...
         NavHostFragment navHostFragment = (NavHostFragment) getSupportFragmentManager().findFragmentById(R.id.nav_host_fragment);
         assert navHostFragment != null;
-        NavigationUI.setupWithNavController(mBinding.bottomNavigation, navHostFragment.getNavController());
+        NavController navController = navHostFragment.getNavController();
+        NavigationUI.setupWithNavController(mBinding.bottomNavigation, navController);
+        NavigationUI.setupWithNavController(mBinding.toolbar, navController);
+        mBinding.toolbar.setTitle(" ");
+
+        navController.addOnDestinationChangedListener((controller, destination, arguments) -> {
+            if (destination.getId() != R.id.authenticationFragment) {
```
Comment (fodeguirassy) [inline app/src/main/java/com/picone/go4lunch/presentation/ui/MainActivity.java:50]: This block of code can be factorized through a private method.

File: app/src/main/res/values/styles.xml [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <resources>
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
```
Comment (fodeguirassy) [inline app/src/main/res/values/styles.xml:3]: Switch to Material Design

File: app/src/main/java/com/picone/go4lunch/presentation/ui/MainActivity.java (hunk 4 above)
Comment (JustAnOtherCoder13) [reply app/src/main/java/com/picone/go4lunch/presentation/ui/MainActivity.java:50]: ok done

File: app/src/main/res/values/styles.xml (hunk 5 above)
Comment (JustAnOtherCoder13) [reply app/src/main/res/values/styles.xml:3]: ok done

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (JustAnOtherCoder13) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (JustAnOtherCoder13) [reply app/src/main/res/layout/activity_main.xml]: done

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (JustAnOtherCoder13) [reply app/src/main/res/layout/activity_main.xml]: done

**Pull request closed without merging**